src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -23,24 +23,23 @@ public FindCommand parse(String args) throws ParseException {
         String trimmedArgs = args.trim();
         if (trimmedArgs.isEmpty()) {
             throw new ParseException(
-                String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
```

Comment:
Is this indentation correct? It looks like double of what it should be but not too sure

Reply:
Yeah, I think I have to make it eight spaces because it is part of the ParseException, and then I'm moving it to the next line like a wrapped line.

From the coding standard website:
Indentation for wrapped lines should be 8 spaces (i.e. twice the normal indentation of 4 spaces) more than the parent line.

setText("Long line split"
        + "into two parts.");
if (isReady) {
    setText("Long line split"
            + "into two parts.");
}

Reply:
okie!